doc/api/buffer.md

```diff
@@ -38,11 +38,11 @@ const buf3 = Buffer.allocUnsafe(10);
 // Creates a Buffer containing [0x1, 0x2, 0x3].
 const buf4 = Buffer.from([1, 2, 3]);
 
-// Creates a Buffer containing ASCII bytes [0x74, 0x65, 0x73, 0x74].
-const buf5 = Buffer.from('test');
-
 // Creates a Buffer containing UTF-8 bytes [0x74, 0xc3, 0xa9, 0x73, 0x74].
-const buf6 = Buffer.from('tést', 'utf8');
+const buf5 = Buffer.from('tést');
+
+// Creates a Buffer containing ASCII bytes [0x74, 0xe9, 0x73, 0x74].
```

Comment:
ASCII encoding uses only the bytes up to 127, and it can’t encode characters such as `é` at all. In some places of the API – like `Buffer.from(…, 'ascii')` – Node just treats `ascii` like `latin-1` for performance reasons, where `é` maps to `0xe9`. That doesn’t always work, though; for example:

```js
> Buffer.from('tést', 'ascii').toString('ascii')
'tist'
```

If you want to use this example, I’d suggest going with something like `Latin-1 bytes`, and use `Buffer.from(…, 'latin-1')`.